client/my-sites/marketing/main.jsx

```diff
@@ -123,6 +126,22 @@ export const Sharing = ( {
 		titleHeader = translate( 'Integrations' );
 	}
 
+	// Determine the right type of upsell according to the plan.
+	// For the Free plan and the Starter plan, we want to upsell the no-ads add-on.
+	// Otherwise, it upsells the Pro plan.
+	let upsellNudgeTitle = translate( 'No ads with WordPress.com Pro' );
+	let upsellNudgeDesc = translate( 'Prevent ads from showing on your site.' );
+	let upsellNudgeHref;
+
+	if (
+		isStarterPlanEnabled() &&
+		( isFreePlan( currentPlanSlug ) || isStarterPlan( currentPlanSlug ) )
+	) {
+		upsellNudgeTitle = translate( 'Remove ads from your site with the No-Ads add-on' );
+		upsellNudgeDesc = translate( 'Click here to add to cart and continue.' );
```

Comment:
"Click here" should probably be avoided... So the above could at least be shortened.

But is it really useful at all?  If we have nothing more to say about the add-on itself, then could the description just be left blank instead?

Reply:
Changed. Looks cleaner now:
<img width="320" alt="Screenshot on 2022-05-23 at 13-13-56" src="https://user-images.githubusercontent.com/2749938/169804109-69c2d234-9c3c-4f90-b3af-1ba3fedeee2d.png">